### PR TITLE
create programs, and refactored components

### PR DESCRIPTION
**Pull request by andrewjcasal**

### CODE REVIEW

File: src/Components/DragFile.js [hunk 1]
```diff
@@ -1,13 +1,30 @@
...
-    </div>
-    <p>{props.label}</p>
-  </div>
-)
+class DragFile extends React.Component {
+  constructor() {
+    super()
+    this.state = { files: [] }
+  }
+
```
Comment (mKleinCreative) [inline src/Components/DragFile.js]: If you're not using the constructor, you can just set the state like so,

```
state = {
  files: []
}
```

File: src/Components/DragFile.js [hunk 2]
```diff
@@ -1,13 +1,30 @@
...
+  onDrop(files) {
+    this.setState({
+      files
+    });
+  }
+
```
Comment (mKleinCreative) [inline src/Components/DragFile.js]: Where is onDrop being called? or is that a WIP

File: src/Pages/CreateProgram.js [hunk 3]
```diff
@@ -36,38 +37,30 @@ const CreateProgram = () => (
             </div>
           </div>
           <div className="row">
-            <div className="col-md-3">
-              <input type="text" placeholder="Registration Open" className="datetimepicker" id='datetimepicker1' />
-            </div>
-            <div className="col-md-3">
-              <input type="text" placeholder="Open Time" />
+            <div className="col-md-6">
```
Comment (mKleinCreative) [inline src/Pages/CreateProgram.js]: I wonder if there's a way for us to cut down on size, by doing something likeee...
```
<TableInfo size={'col-md-6'} type={DatePicker} placeHolder="Registration Open Date/Time" includeTime />
```
and just have a bunch of table inputs like that, that we can easily add or make a template out of.

File: src/Pages/CreateProgram.js [hunk 4]
```diff
@@ -36,38 +37,30 @@ const CreateProgram = () => (
...
-              <input type="text" placeholder="Registration Close" class="datetimepicker" id='datetimepicker3' />
-            </div>
-            <div className="col-md-3">
-              <input type="text" placeholder="Close Time" />
+            <div className="col-md-6">
+              <Datepicker placeholder="Registration Close Date/Time" includeTime/>
             </div>
             <div className="col-md-3">
-              <input type="text" placeholder="Event End" className="datetimepicker" id='datetimepicker4' />
+              <Datepicker placeholder="Event End" />
```
Comment (mKleinCreative) [inline src/Pages/CreateProgram.js]: see above for these as well.

File: src/Components/DragFile.js (hunk 1 above)
Comment (andrewjcasal) [reply src/Components/DragFile.js]: This is a work in progress. I would soon be building this out to manage storing the file. Keep for now?

File: src/Components/DragFile.js (hunk 2 above)
Comment (andrewjcasal) [reply src/Components/DragFile.js]: I'm using a plugin called "DropZone" look it up. Keep for now though?

File: src/Pages/CreateProgram.js (hunk 3 above)
Comment (andrewjcasal) [reply src/Pages/CreateProgram.js]: Is there a standard/convention for this? Seems like cutting down on 2 lines might not be worth it's value. Would love to hear your case.

Comment (mKleinCreative) [reply src/Pages/CreateProgram.js]: to me, it'd be more about customization, and added readability, By creating this modular component, we could then release it into the wild and take down an exponential amount of lines of code, and have the ability to render an input with a size where ever we wanted too, Which occurs a lot inside of this app.

File: src/Components/DragFile.js (hunk 2 above)
Comment (mKleinCreative) [reply src/Components/DragFile.js]: Dropzone is pretty sweet,  keep it.

**Pull request merged**